src/rules/operator-no-unspaced/__tests__/index.js

```diff
@@ -2493,6 +2493,23 @@ testRule(rule, {
   syntax: "scss",
   skipBasicChecks: true,
 
+  accept: [
+    {
+      code: "@import url('10px +2');",
```

Comment:
Could this be changed to be an actual valid url instead of just adding some math inside of a string?

Reply:
Done (note the existing reject cases include `url(10px +2)`).